fastplotlib/graphics/linecollection.py

```diff
@@ -113,3 +117,35 @@ def __init__(
             )
 
             self.add_graphic(lg, reset_index=False)
+
+    def _set_feature(self, feature: str, new_data: Any, indices: Union[int, range]):
+        if not hasattr(self, "_previous_data"):
+            self._previous_data = {}
+        elif hasattr(self, "_previous_data"):
+            self._reset_feature(feature)
+        #if feature in # need a way to check if feature is in
+        feature_instance = getattr(self, feature)
+        if indices is not None:
+            previous = feature_instance[indices].copy()
+            feature_instance[indices] = new_data
```

Comment:
for graphic collections the indices will refer to graphic indices, not vertex indices like for simple graphics.:

try:

```python
coll_feature = getattr(self[indices], feature)
coll_feature._set(new_data)
```